ivy_models/googlenet/layers.py

```diff
@@ -0,0 +1,185 @@
+import ivy
+
+
+class ConvBlock(ivy.Module):
```

Comment:
a small request, would you mind renaming the layers here to `GoogleNet<Layer>` for convention purposes? thanks!